lib/lita/handlers/imgflip.rb

```diff
@@ -10,7 +10,7 @@ class Imgflip < Handler
 
       route %r{(Y U NO) (.+)}i,                                    :meme_y_u_no,             help: { "Y U NO..." => "generates Y U NO meme"}
       route %r{(I DON'?T ALWAYS .*) (BUT WHEN I DO,? .*)}i,        :meme_i_dont_always,      help: { "I DON'T ALWAYS .. BUT WHEN I DO, .." => "generates The Most Interesting Man in the World meme"}
-      route %r{(.*)(O\s?RLY\??.*)}i,                               :meme_orly,               help: { "..O RLY.." => "generates O RLY meme" }
+      route %r{(.*) (O\s?RLY\??.*)}i,                              :meme_orly,               help: { "..O RLY.." => "generates O RLY meme" }
```

Comment:
(hah, i'm giggling to myself that we're having a such a serious conversation over something so decidedly _not_ serious :joy:)